### PR TITLE
Fix/localinstall [SBK-348]

### PR DESCRIPTION
**Pull request by alexisnsns**
Import apeworx/apepay and apeworx/apepay-react in the demo app by doing a local install instead of doing local imports

### CODE REVIEW

**Review by fubuloubu — APPROVED** [review]

File: ui/app/src/App.tsx [hunk 1]
```diff
@@ -94,7 +98,7 @@ function App() {
 
   // Fetch logs starting from this block
   // TODO: find a way to get the SM deployment block
-  const fromBlock = 4615000n;
+  const fromBlock = config.fromBlock ? BigInt(config.fromBlock) : undefined;
```
Comment (fubuloubu) [inline ui/app/src/App.tsx:101]: Wouldn't null be better?

**Review by fubuloubu — APPROVED** [review]

File: ui/app/src/App.tsx (hunk 1 above)
Comment (alexisnsns) [reply ui/app/src/App.tsx:101]: The expected type in getContractEvents is `fromBlock?: bigint | BlockTag | undefined`, TS doesn't like it when i replace it with null

Comment (fubuloubu) [reply ui/app/src/App.tsx:101]: oh, is that what `?` means? a type or `undefined`?

Comment (alexisnsns) [reply ui/app/src/App.tsx:101]: yes that's why the '?' in the optional param; if you still pass an optional param when calling the function (as in our case), you'll need to pass it as undefined

File: ui/lib/UpdateStream.tsx [hunk 2]
```diff
@@ -160,7 +160,7 @@ const UpdateStream: React.FC<UpdateStreamProps> = (props) => {
             disabled={isButtonDisabled}
             className="update-stream-button"
           >
-            Fund stream
+            Fund
```
Comment (fubuloubu) [inline ui/lib/UpdateStream.tsx:163]: I think `Add Time` might be nicer

**Review by fubuloubu — APPROVED** [review]

**Review by fubuloubu — APPROVED** [review]

**Pull request merged**